storage/block_devices_ops.go

```diff
@@ -541,21 +545,593 @@ func partitionUsingParted(bd *BlockDevice, dryRun *[]string, wholeDisk bool) err
 	return nil
 }
 
+// removePhysicalvolume actually performs the operation of removal of a physical volume
+func removePhysicalvolume(bd *BlockDevice, dryRun *DryRunType) error {
+	if bd.FsType != "LVM2_member" {
+		return errors.Errorf("Block Type is not physical volume")
+	}
+
+	args := []string{
+		"pvremove",
+		bd.GetMappedDeviceFile(),
+	}
+
+	if dryRun != nil {
+		*dryRun.CurrentTargetResults = append(*dryRun.CurrentTargetResults,
+			utils.Locale.Get("Remove physical volume: %s", bd.Name))
+	} else {
+		log.Info("Proceeding to remove physical volume: %s", bd.GetMappedDeviceFile())
+		err := cmd.RunAndLog(args...)
+		if err != nil {
+			return errors.Wrap(err)
+		}
+	}
+
+	return nil
+}
+
+/*
+checkVolumeGroupSpan is check operation to see if all PVs in VG in the entire system
+can be found in scans.ScannedPvList for specific disk which we computed earlier
+If not Found, then the PV belongs to another disk and that becomes a red flag
+*/
+func checkVolumeGroupSpan(pvDisplayOutput *bytes.Buffer, ForceDestructive bool, volumeGroup string,
+	dryRun *DryRunType, scans *preScanResults) error {
+	log.Debug("checkVolumeGroupSpan: Running with ForceDestructive:%v in %s mode", ForceDestructive, findMode(dryRun))
+
+	if ForceDestructive && dryRun == nil {
+		return nil
+	}
+
+	var pVList = []string{}
+	for _, line := range strings.Split(pvDisplayOutput.String(), "\n") {
+		fields := strings.Split(line, ":")
+		if len(fields) == 12 {
+			if strings.TrimSpace(fields[1]) == volumeGroup {
+				pVList = append(pVList, strings.TrimSpace(fields[0]))
+			}
+		}
+	}
+
+	sort.Strings(pVList)
+	var otherDisks = []string{}
+	for _, pv := range pVList {
+		found := false
+		for _, pvblk := range scans.ScannedPvList {
+			if pv == pvblk.GetMappedDeviceFile() {
+				found = true
+				break
+			}
+		}
+
+		if !found {
+			otherDisks = append(otherDisks, pv)
+		}
+	}
+
+	if len(otherDisks) > 0 {
+		if dryRun != nil {
+			if len(*dryRun.OtherMediaResults) == 0 {
+				*dryRun.OtherMediaResults = append(*dryRun.OtherMediaResults, "Effect on Other Disks:")
```

Comment:
You have a locale for this phrase, but you need to call it here.
```suggestion
				*dryRun.OtherMediaResults = append(*dryRun.OtherMediaResults, utils.Locale.Get("Effect on Other Disks:"))
```

Reply:
Done